runtimes/go/beta/errs/error.go

```diff
@@ -167,9 +193,10 @@ func (e *Error) ErrorMessage() string {
 	var next error = e.underlying
 	for next != nil {
 		var msg string
-		if e, ok := next.(*Error); ok {
-			msg = e.Message
-			next = e.underlying
+		var ee *Error
+		if errors.As(next, &ee) {
+			msg = ee.Message
+			next = ee.underlying
```

Comment:
This will skip the error message of any intermediate errors in the chain between this error and the next wrapped `*Error`. I think that's undesirable, and that we should probably revert this part of the PR

Reply:
good catch! have updated